### ISSUE TITLE
Angle at top of /acute /grave and relatives

### ISSUE THREAD

**Issue by tphinney** [tphinney/science-gothic#157]
I've been considering angling the tops of /acute /grave and relatives. Currently trialing this in the combining acute. I have decided to move forward with this change.

This could be done in combination with the change in contrast handling for the same accents plus /circumflex, #155

**Comment by servantb** [comment]
After sitting with it a while, comparing to the original (horizontal tops) style, and seeing it in context with other diacritics (circumflex and caron), I think the original horizontal tops work better.

Having the horz tops may also help add harmony to the diacritics if you apply the alternative contrast style to the acute and grave (issue #155).

**Comment by tphinney** [comment]
Did a combined experiment to check out this and #155 
https://recordit.co/mU1czaLjJo
https://recordit.co/5zE1fqYpP4

**Comment by tphinney** [comment]
Although I like the look of this, still, I am going to tackle #155 first, and then consider this separately.

**Comment by tphinney** [comment]
After much review, I decided ... **@servantb is right**! This does not work as well as the original flat top.

I think it looked good enough in the non-contrast weights, and in the single acute. Even arguably a slight improvement to my eyes. **But** it combined fairly poorly with the new triangular shape in the high-contrast masters, and that was a definite win for a real problem, so I don’t want to do this instead of that. And the slanted tops looked **_very_** bad in the double acute (hungarumlaut) used in Hungarian. So ... that’s a no from me, after all.

I have rolled it back for my next check-in tonight.
